CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Various fixes [Theodor]
```

Comment:
I still feel we should detail here a bit. Like, settings CONFIG_KEYS now required by docker 17.03 or whatever and undefining obsolete INITRAMFS_TASK